handlers.go

```diff
@@ -1484,7 +1484,7 @@ func (s *Server) handleGetContentByCid(c echo.Context) error {
 		return err
 	}
 
-	var out []getContentResponse
+	out := make([]getContentResponse, 0)
```

Comment:
added this change so when a cid is not estuary, it does not return null, but an empty array

<img width="922" alt="Screenshot 2022-07-21 at 10 01 59" src="https://user-images.githubusercontent.com/13554411/180175007-d4a63ff1-c5c5-499c-ac31-e10bc1f14094.png">

VS

![Uploading Screenshot 2022-07-21 at 10.03.57.png…]()